ingest_python_scripts/nara/copy_nara.py

```
@@ -0,0 +1,302 @@
+import base64
+import json
+import re
+import shlex
+import subprocess
+import sys
+import time
+
+# ---------- config ----------
+def _load_dotenv():
+    cfg = {}
+    env_file = os.path.normpath(
```

Ruff (0.15.12)

[error] Undefined name `os`

(F821)